server/index.js

```diff
@@ -19,6 +19,9 @@ app.use(express.static(path.join(__dirname, '../client/dist')));
 
 // routes/controllers will utilize client requests and communicate with the API here.
 app.get('/product', controllers.getProduct);
+app.get('/products', controllers.getAll);
+app.get('/products/:product_id/related', controllers.getRelated);
```

Comment:
Just a small comment, I don't think the route has to change depending on product_id. It can just be a fixed route for all as long as the product_id in the controllers URL changes.

Reply:
@boyiq I think it has to stay as variable as I am using the URL params in the in the requestURL of the getRelated controller function? That way, whatever is passed in from the client gets added to req.params? Happy to hop on a call to discuss tho.

Reply:
@cmfung Right, it is the request params that pass in the product id. My understanding is as long as you set up the params in your get request to localhost, you'll have access to it later. This is just a route to let app know which function in the controllers to execute.